apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -581,6 +771,7 @@ impl Webserver {
             current_version: project.cur_version().to_string(),
             configured_producer: producer,
             is_prod: project.is_production,
+            metrics: metrics.clone(),
```

Comment:
Hum I see why you are cloning here, the service takes ownership of this when it probably should not. instead of passing the metrics object, you could try and pass in the `send_data` so instead so that you don't have to clone and create a duplicate of your internal metrics store